### PR TITLE
feat:search and category filter

### PR DESCRIPTION
**Pull request by shristabaruah**
- search functionality
- filters by categories

### CODE REVIEW

**Review by Ayushi20-19 — APPROVED** [review]
- The UI is cool
- search and filter working fine
- variable naming is good

File: src/components/Filter/filter.module.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.filter_container {
+  display: flex;
+  top: 5rem;
+  position: absolute;
+  gap: 2rem;
+  padding-left: 3rem;
+  flex-wrap: wrap;
+}
+.filter_container button {
```
Comment (Jay-Gadhiya) [inline src/components/Filter/filter.module.css:9]: Suggestion : When you write more than one selector then be sure about specificity and inheritance

File: src/pages/Home/home.jsx [hunk 2]
```diff
@@ -1,26 +1,62 @@
...
+    }
+    return videos;
+  };
+
+  const filterHandler = (videos, filterCategory) => {
+    if (filterCategory) {
+      const categories = videos.filter(
+        (video) => video.category === filterCategory
+      );
+      return categories;
```
Comment (Jay-Gadhiya) [inline src/pages/Home/home.jsx:30]: you can use the ternary operator, i will convert into one line code

**Review by Jay-Gadhiya — APPROVED** [review]
- Otherwise code looks clean
- functionality working fine

**Pull request merged**